apps/web/modules/auth/login/components/login-form.tsx

```diff
@@ -40,6 +40,9 @@ interface LoginFormProps {
   oidcDisplayName?: string;
   isMultiOrgEnabled: boolean;
   isSSOEnabled: boolean;
+  SAMLSSOEnabled: boolean;
```

Comment:
correct camel case would be `samlSsoEnabled`, but we should follow the same pattern as in the other variables with `is` prefix:
`isSamlSsoEnabled`

Please also check other occurrences in the code.